llpc/tool/amdllpc.cpp

```diff
@@ -328,6 +328,10 @@ cl::opt<bool> AssertToMsgBox("assert-to-msgbox", cl::desc("Pop message box when
 cl::opt<bool> EnableInternalRtShaders("enable-internal-rt-shaders",
                                       cl::desc("Enable intrinsics for internal RT shaders"),
                                       cl::init(false));
+
+cl::opt<bool> GpuRtUseDumped("gpurt-use-dumped", cl::desc("Use the GPURT shader library that was dumped with the pipeline dump (may fail due to incompatibility)"));
+
+cl::opt<std::string> GpuRtLibrary("gpurt-library", cl::desc("Use the GPURT shader library from the given file"));
```

Comment:
Where does one get such a dumped GPURT library file from? Is there an option to dump the one built into LLPC (or the one in a pipeline dump) into a file?

Reply:
Pipeline dumps will contain one by default.

Though the main anticipated use case would be that somebody is iterating on GPURT itself and takes the .spv directly from that build.